agent/structs/connect_proxy_config.go

```diff
@@ -469,7 +468,7 @@ func (p *ExposePath) ToAPI() api.ExposePath {
 }
 
 // Finalize validates ExposeConfig and sets default values
-func (e *ExposeConfig) Finalize(l *log.Logger) {
+func (e *ExposeConfig) Finalize() {
```

Comment:
This function does not log anything, not sure why a logger was even passed in here.